### DISCUSSION TITLE
v1.28.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1031] [General]
## Features
- The events chart now updates when an event type filter is selected
![Animation4](https://user-images.githubusercontent.com/477975/158937183-f5152d3f-e9b6-404a-bace-114769a97815.gif)
- Added `data-css-events` tracker parameter to disable umami style css events. See https://umami.is/docs/track-events
- Tracker now uses navigator.sendBeacon() to ensure data is sent
- Subdomains on your domain are now also filtered from the referrers table

## Updates
- Added postbuild step which collects completely anonymous telemetry data. To disable set the environment variable `TELEMETRY_DISABLED`
- Lots of minor CSS styling changes
- Updated Prisma to `3.11.0`

## Fixes
- Fixed issue with Firefox not sending events
- Fixed issue with `FORCE_SSL` env variable causing infinite redirects

A huge thanks to all the contributors on this release! 🙏@Klooven

<hr /><em>This discussion was created from the release <a href='https://github.com/mikecao/umami/releases/tag/v1.28.0'>v1.28.0</a>.</em>

**Comment by H0rn0chse** [comment]
Hi thanks for the update 🚀 
Just a small note: I guess you've misspelled `TELEMETRY_DISABLE` to `TELEMETRY_DISABLED` in the release notes.

**Reply by mikecao** [reply]
Thanks, fixed!

**Comment by fre-ben** [comment]
Hey, quick question: Currently running v.1.23.0 in a docker-container on a cloud server.
Is updating to v.1.28.0 possible without any issues, or will updating break anything or erase current data?

**Reply by mikecao** [reply]
There should be no issues with updating since there have been no data changes, only changes to the UI.

**Reply by fre-ben** [reply]
Thanks for the quick reply and keep up the good work :)
